src/express.ts

```diff
@@ -67,3 +67,45 @@ export const serve: ServeHandler = (nameOrInngest, fns, opts) => {
 
   return handler.createHandler();
 };
+
+function getBody(req: Request): Record<string, unknown> {
+  const body: unknown = req.body;
+
+  if (body === undefined) {
+    throw new Error("missing request body");
+  }
+
+  if (typeof body !== "object" || body === null || Array.isArray(body)) {
+    throw new Error("body is not an object");
```

Comment:
Could this be a `throw new Error(prettyError(...))` which directs users to https://www.inngest.com/docs/sdk/serve#framework-express so they can fix the issue for themselves?